Makefile

```diff
@@ -59,6 +60,9 @@ COMPONENTS_USAGE_STATS_FILES = $(shell \
 		-type f \
 		\( -name '*.js' -or -name '*.jsx' \) \
 )
+COMPONENTS_PROPTYPE_FILES = $(shell \
+	find client -name 'index.jsx' -or -name 'example.jsx' \
```

Comment:
Actually, I think having data on components outside `blocks` and `components` will be useful for future README generation. So ignore this :)